extensions/widgets/navbar/navbar.lcb

```diff
@@ -248,18 +269,26 @@ property desiredHeight	get getDesiredHeight
 metadata desiredHeight.user_visible is "false"
 
 /*
-Syntax: set the <opaqueBackground> of <widget> to {true | false}
-Syntax: get the <opaqueBackground> of <widget>
+Syntax: set the <backgroundOpacity> of <widget> to <pOpacity>
+Syntax: get the <backgroundOpacity> of <widget>
 
-Summary: Whether the background of the navbar is drawn opaque or not
+Summary: The opacity of the background of the navbar
 
-Description:
-Use the <opaqueBackground> property to determine whether objects underneath the navbar are visible or not.
+Parameters:
+pBackgroundOpacity(enum): The opacity of the background
```

Comment:
This has to match the name of the parameter in the syntax otherwise the docs builder complains